intercom_flutter/README.md

```diff
@@ -145,3 +145,63 @@ Add the below script inside body tag in the index.html file located under web fo
 - [ ] handlePush
 - [ ] displayCarousel
 - [ ] displayHelpCenterCollections
+
+## Using of intercom keys with --dart-define
```

Comment:
Wording here should be `Use of intercom...` or `Using intercom...`, and --dart-define should be wrapped with `